apps/explorer/src/components/module/module-functions-interaction/ModuleFunction.tsx

```diff
@@ -66,6 +69,14 @@ export function ModuleFunction({
             return result;
         },
     });
+    const executeFunctionError = execute.error
+        ? (execute.error as Error).message
+        : false;
+    const allValues = useWatch({ control });
+    const mutationReset = execute.reset;
+    useEffect(() => {
+        mutationReset();
```

Comment:
This seems super weird, why are we doing this?

Reply:
This is done to clear the result of the tx execution when the user starts to enter new arguments

Reply:
Yeah I can do that, would simplify things as well

Reply:
removed